include/seahorn/HornClauseDBTransf.hh

```diff
@@ -1,15 +1,19 @@
-#ifndef _HORN_CLAUSE_DB_TRANSFORMATIONS__H_
-#define _HORN_CLAUSE_DB_TRANSFORMATIONS__H_
+// Horn Clause DB Transformations
+#pragma once
 
 #include "seahorn/HornClauseDB.hh"
 
 namespace seahorn
 {
 
-  // Ensure all horn clause heads have only variables
-  void normalizeHornClauseHeads (HornClauseDB &db);
+// Ensure all horn clause heads have only variables
+void normalizeHornClauseHeads (HornClauseDB &db);
 
-}
+// Remove arguments of type FiniteMap from Horn Clauses
+void removeFiniteMapsArgsHornClausesTransf(HornClauseDB &db, HornClauseDB &tdb);
```

Comment:
Why not to have just a single transformation called `removeFiniteMaps`?

Reply:
I guess it was for debugging purposes, I will merge them